api-docs/rst/dev-guide/concepts.rst

```diff
@@ -31,9 +34,11 @@ Autoscale functions by linking these services:
 
 -  Load-balanced Cloud Servers. Learn more at `Load Balancers Getting Started Guide`_.
 
+-  Rackspace Cloud Orchestration. Learn more at `Cloud Orchestration Getting Started Guide`_.
+
 You can set up a schedule for launching Autoscale or define an event
 that triggers a webhook. You can also specify a minimum and maximum
-number of cloud servers for your scaling group, the amount of resources
+number of scaling units for your scaling group, the amount of resources
 you want to increase or decrease, and policies based on percentage or
 real numbers.
 
```

Comment:
You can also specify a minimum and maximum number of scaling units for your scaling group, and specify an integer or percentage value to increase or decrease the number of resources.